src/styles/blocks/calendar.scss

```
@@ -0,0 +1,57 @@
+.calendar {
+  &__day {
+    position: relative;
+
+    @include daySize ($calendar-day-size);
+    border: 1px solid #000;
+
+    background-color: $calendar-day-color;
+    transition: 0.5s, transform 0.5s;
+
+    &:hover {
+      background-color: #ffc0cb;
+
+      cursor: pointer;
+      transform: translateY(-20px);
+    }
+  }
+}
+
+@for $index from 28 through 31 {
+  .calendar--month-length_#{$index} {
+    @for $day-index from $index + 1 through 32 {
+      .calendar__day:nth-child(#{$day-index}) {
+        display: none;
+      }
+    }
+  }
```

no need for double loop, you can use `nth-child` for it
```suggestion
  .calendar--month-length_#{$index} {
      .calendar__day:nth-child(n + #{$day-index} + 1) {
        display: none;
      }
    }
  }
```